OpenOversight/app/models.py

```diff
@@ -135,6 +139,8 @@ def race_label(self):
                 return label
 
     def gender_label(self):
+        if self.gender is None:
+            return 'Data Missing'
```

Comment:
Differentiating between missing data on our side and the user not knowing or caring is a great idea! And I think it can make some things much clearer. I like displaying `'Data Missing'`, the other options I see are either leaving it blank or skipping the display in the table altogether.
However, I don't think we need to decide on one way now, so this is definitely fine. Once we did similar work on other demographic data we always have the option of going back and changing it to a different behavior.

Reply:
yeah, eventually I'd like to see a unified approach to these issues across demographic categories, and to make sure the UI makes very clear what's going on to the user.